src/content/guides/ecommerce.mdx

```diff
@@ -0,0 +1,31 @@
+---
+title: eCommerce
+description: How to process payments using Centrapay's eCommerce Platform.
+nav:
+  path: Reference/Merchant Integrations
+  order: 2
+---
+
+Centrapay enables businesses to process payments for connected Centrapay assets online. In order to process payments businesses must be able to integrate with one of our eCommerce payment flows.
+
+## Onboarding
```

Comment:
Suggestion: You could make this section more concise so that it's easier to read

Something like "Please contact Centrapay to get your business configured for ecommerce. Centrapay will provide you with API key to request payment(link)

Centrapay will configure your business to accept test payments before configuring you to accept live payments. Accepting live payments requires you to meet our certification requirements.

Reply:
Updated it to match your suggestion. :)